.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
   - '2.7'
```

Comment:
end of life

Reply:
Yes, but still being widely used, it does not harm anyone to keep testing it. I might remove testing later next year. Thanks for the reminder though!